### ISSUE TITLE
Not updating in realtime

### ISSUE THREAD

**Issue by gableoley** [Vinyl-Davyl/talku-talku-v3#1]
When you send a message to some one it does not get receive in realtime.

**Comment by Vinyl-Davyl** [comment]
yeah, noticed that too. would fix that and push some updates soon. as soon as I get working again!

you could checkout the v2 of "talku talku" on my github here https://github.com/Vinyl-Davyl/talku-talku-v2 for the syntax and logic you need for that. 

Incase it's something urgent you need. would also welcome a PR as a contribution if you find a precise way around that. would review on my end.
